projects/packages/search/src/dashboard/class-initial-state.php

```diff
@@ -82,7 +82,7 @@ public function get_initial_state() {
 				'instant_search_enabled' => $this->module_control->is_instant_search_enabled(),
 			),
 			// phpcs:ignore WordPress.Security.NonceVerification.Recommended
-			'features'        => array_map( 'sanitize_text_field', explode( ',', $_GET['features'] ) ),
+			'features'        => array_map( 'sanitize_text_field', explode( ',', isset( $_GET['features'] ) ) ? $_GET['features'] : '' ),
```

Comment:
Good catch 👍 We can probably skip the `explode` if $_GET['features'] isn't set?

Reply:
good point addressed in 8800b5455d